### DISCUSSION TITLE
WebXR Support

### DISCUSSION THREAD

**Discussion by EricMoerthVis** [visgl/deck.gl#7972] [Ideas]
I was wondering with Deck.gl or Luma.gl will have a WebXR support in the future?

**Comment by ibgreen** [comment]
We'd love to add it but it is a good bit of work. Unless someone new rises to the task, I suspect the focus for now will be to complete the port to WebGPU.

There was an attempt a couple of years ago to get stereoscopic rendering and headset interaction in luma.gl via WebVR (the Display classes) but all those APIs have been deprecated so a new effort would be needed.

**Comment by willemhelmet** [comment]
Checking in on this thread -- has there been any progress towards enabling WebXR support with Deck.gl?
